src/components/SimulatorArea.tsx

```diff
@@ -98,6 +106,13 @@ export class SimulatorArea extends React.Component<SimulatorAreaProps> {
   };
 
   render() {
+    if (this.state.loading) {
+      return (
+        <Container >
+          <Loading message='Initializing Robot...'/>;
```

Comment:
Can we show something more general like "Loading simulator..." or "Initializing scene..."?

Reply:
Sure! I just thought this would be a fun message, but I think either of those also work